exporter/containerimage/attestations.go

```diff
@@ -172,6 +176,8 @@ func newFileLayerFinder(target cache.ImmutableRef, remote *solver.Remote) (fileL
 //
 // find is not concurrency-safe.
 func (c *fileLayerFinder) find(ctx context.Context, s session.Group, filename string) (cache.ImmutableRef, *ocispecs.Descriptor, error) {
+	filename = filepath.Clean("/" + filename)
```

Comment:
isn't this supposed to be `filepath.Join("/", filename)`

Reply:
Thinking about the use of `filepath` as well. I think these paths should be unix normalized, but that can be follow-up when wcow becomes more realistic.